### PR TITLE
added how packs work diagram, removed broken namespace reference

### PR DESCRIPTION
**Pull request by punit-codaio**

### CODE REVIEW

**Comment by punit-codaio** [comment]
for images in the documentation, should i add this into the repo or allow GH to host these? I did latter, can fix if needed

**Comment by jonathan-codaio** [comment]
> for images in the documentation, should i add this into the repo or allow GH to host these? I did latter, can fix if needed

How did you upload the image?

**Comment by punit-codaio** [comment]
I dragged it into the online text editor on GH... it's thus served off of the `user-images.githubusercontent.com` domain

**Review by jonathan-codaio — APPROVED** [review]
I guess that works.

File: README.md [hunk 1]
```diff
@@ -60,20 +59,21 @@ behalf when the Pack is used in a doc. This is different from our API, where we
...
+**Use the Pack in a Doc**
 
 3. **Doc Makers/Editors install the Pack in their doc**
 4. **They invoke formulas from the doc**: When any formulaic element (e.g. formula, sync table, column format, button) is invoked, the request is sent to the Coda backend
 5. **Coda runs the formula**: we retrieve your Pack build and run it for you in the Packs Secure Execution Environment. The formulas are run _only with the inputs provided to it_; each call to a formula is independent as formulas are state-less. Thus, Packs cannot rely on data that changes and persists between individual calls to a formula.
 6. **If applicable, external API called** by the Execution Environment. All API calls will originate from our servers to protect authentication credentials, which we store on our servers and never send to clients.
 7. **Return data to doc**: the results (or errors) from the execution are sent back to the doc.
 
-TODO: add diagram
+![packs execution](https://user-images.githubusercontent.com/79715674/125986426-7dc52e30-4bc6-4e18-98a9-ba5a293b6f6a.png)
```
Comment (huayang-codaio) [inline README.md:75]: very cool!

Comment (punit-codaio) [reply README.md:75]: @alicia-codaio 's work making that so readable!

**Pull request merged**